docs/aca/09-aca-autoscale-keda/index.md

```diff
@@ -1,9 +1,7 @@
 ---
-title: Module 9 - ACA Auto Scaling with KEDA
-has_children: false
-nav_order: 9
-canonical_url: 'https://bitoftech.net/2022/09/22/azure-container-apps-auto-scaling-with-keda-part-11/'
+canonical_url: https://bitoftech.net/2022/09/16/use-bicep-to-deploy-dapr-microservices-apps-to-azure-container-apps-part-10/
```

Comment:
this should be: https://bitoftech.net/2022/09/22/azure-container-apps-auto-scaling-with-keda-part-11/